commit 7f72b573c50b663be5fdccee7d4fb7f2187a7493 (refs/changes/63/663/2)
Author: Richard Kiene <richard.kiene@joyent.com>
Date:   2016-10-13T18:55:30+00:00 (3 years ago)
    
    CNS-167 Upgrade changefeed to 1.2.0 which contains the fix for AGENT-1041
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/package.json b/package.json
index 9992d5e..abef833 100644
--- a/package.json
+++ b/package.json
@@ -25,7 +25,7 @@
   "dependencies": {
     "assert-plus": "0.1.5",
     "bunyan": "1.5.1",
-    "changefeed": "1.1.9",
+    "changefeed": "1.2.0",
     "cueball": "1.0.2",
     "dashdash": "1.10.1",
     "deep-equal": "1.0.1",
